src/test/java/org/broadinstitute/hellbender/engine/filters/CountingVariantUnitTest.java

```diff
@@ -0,0 +1,308 @@
+package org.broadinstitute.hellbender.engine.filters;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.*;
+
+public final class CountingVariantUnitTest {
```

Comment:
Sure, I think that this was an error caused by some refactoring/rebase... I can't actually remember....